<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LandlordClass</class>
 <widget class="QMainWindow" name="LandlordClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>840</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>840</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1275</width>
    <height>908</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Landlord</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Resources/logo.ico</normaloff>:/Resources/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="player_A" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>600</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="player_C" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>620</y>
      <width>600</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_A">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_B">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_C">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>600</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="path_A" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="path_B" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>309</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="path_C" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="player_B" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>600</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>-2</y>
      <width>172</width>
      <height>105</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="bidList"/>
   </widget>
   <widget class="QLabel" name="arror_C">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>54</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>46</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="arror_B">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>54</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>46</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="arror_A">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>54</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>46</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="bidCard" native="true">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>0</y>
      <width>191</width>
      <height>134</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="file_path">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>3</y>
      <width>785</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="bg" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>99999</width>
      <height>9999999</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
background-color: rgb(0, 170, 0);</string>
    </property>
   </widget>
   <zorder>bg</zorder>
   <zorder>player_A</zorder>
   <zorder>player_C</zorder>
   <zorder>label_A</zorder>
   <zorder>label_B</zorder>
   <zorder>label_C</zorder>
   <zorder>player_B</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>arror_C</zorder>
   <zorder>bidCard</zorder>
   <zorder>path_C</zorder>
   <zorder>path_A</zorder>
   <zorder>arror_A</zorder>
   <zorder>path_B</zorder>
   <zorder>arror_B</zorder>
   <zorder>file_path</zorder>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Landlord.qrc"/>
 </resources>
 <connections/>
</ui>
